content/post/twitter-hashtags-real-time.md

```diff
@@ -0,0 +1,561 @@
++++
+title = "Identifying Trending Twitter Hashtags in Real-time with Wallaroo"
+date = 2017-11-14T12:00:00-04:00
+draft = false
+slug = "twitter-hashtags-real-time"
+author = "haneemedhat"
+description = "Using Wallaroo to connect to a real-time twitter stream and find trending hashtags"
+tags = [
+    "wallaroo",
+    "twitter",
+    "tutorial",
+    "example"
+]
+categories = [
+    "Tutorial",
+    "Examples"
+]
++++
+
+
+This week we have a guest post written by Hanee' Medhat
+
+Hanee' is a Big Data Engineer, with experience working with massive data in many industries, such as Telecommunications and Banking.
+
+## Overview
+
+One of the primary places where the world is seeing an explosion of data growth is in social media.  [Wallaroo](http://www.wallaroolabs.com/community) is a powerful and simple-to-use open-source data engine that is ideally suited for handling massive amounts of streaming data in real-time.
+
+In this tutorial, I will use Wallaroo to analyze and extract insights from Twitter in real-time and present the results on a dashboard.
+
+First, a little background on Wallaroo.  Wallaroo is a relatively new open-source project that has been getting a lot of attention recently.  Wallaroo Labs, the company behind the project, has shared a good deal of information about their approach and technology. You may have seen some of their recent blog articles on Hacker News.
+
+You can find more information about Wallaroo by visiting [this site](wallaroolabs.com/community).
```

Comment:
This href needs the `https://` at the beginning otherwise it assumes it's a relative path to the blog post uri